gemrb/core/GUI/GameControl.cpp

```diff
@@ -2214,7 +2217,12 @@ bool GameControl::OnMouseUp(const MouseEvent& me, unsigned short Mod)
 		
 		if (overContainer || overDoor || (overInfoPoint && overInfoPoint->Type==ST_TRAVEL && target_mode == TARGET_MODE_NONE)) {
 			// move to the object before trying to interact with it
-			CommandSelectedMovement(p, false, isDoubleClick);
+			Actor* mainActor = GetMainSelectedActor();
+			if (mainActor && (overContainer || target_mode == TARGET_MODE_PICK)) {
```

Comment:
```suggestion
			if (mainActor && overContainer) {
```
Looks like you missed this a couple of days ago.

Reply:
nah i just accidentally put this change in the next commit (that i haven't pushed yet, still waiting this current PR to be merged) 
fixed 10 secs ago
still learning how to use git

Reply:
You're doing fine, don't worry. 👍🏾